spec/models/budget_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+require "./app/models/budget"
+
+describe Budget do
+  it "can be created" do
+    budget = Budget.create(name: "holiday")
+    budget.save!
+
+    expect(budget).to be_a Budget
+  end
```

Comment:
You are testing core Rails functionality here. I don't think this spec is needed.